101-azure-dns-new-zone/metadata.json

```diff
@@ -2,6 +2,6 @@
   "itemDisplayName": "Azure DNS domain hosting example",
   "description": "This template shows how to create a DNS zone within Azure DNS and how to add some record sets to it.",
   "summary": "This template creates a DNS zone within the Azure DNS resource provider.  It then creates a RecordSet containing two DNS A records within that zone.",
-  "githubUsername": "GarethBradshawMSFT",
-  "dateUpdated": "2016-03-21"
+  "githubUsername": "bmoore-msft",
```

Comment:
@bmoore-msft Did you mean to change this? Are you going to own this going forward?